City-of-Helsinki/drupal-hdbt
# SCSS linting

# SCSS linting UHF-X
Fix linter errors that do not change the dist output.

## What was done
* Fixed all linting errors that do not change dist output
* Finetuned linter settings a bit.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-X_scss_linting`
* Run `make drush-cr`

## How to test
* run `npm run build`
* dist should not change
* read code, there should be less linting errors now

## Notice
~Before merging, the clean regions PR should be merged~
